conseil-common/src/main/scala/tech/cryptonomic/conseil/common/config/Platforms.scala

```diff
@@ -137,20 +137,44 @@ object Platforms {
     override val platform: BlockchainPlatform = Bitcoin
   }
 
+  /** configurations to describe a ethereum node */
+  final case class EthereumNodeConfiguration(
+      hostname: String,
+      port: Int,
+      protocol: String,
+      path: String
+  ) {
+    val url = s"$protocol://$hostname:$port$path"
+  }
+
+  /** configurations to describe a ethereum batch fetch */
+  final case class EthereumBatchFetchConfiguration(
```

Comment:
Could this entries be documented, please?

Reply:
Please check it now